python/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ set(ATMMETAFORCEPLUGIN_HEADER_DIR "${CMAKE_SOURCE_DIR}/openmmapi/include")
 set(ATMMETAFORCEPLUGIN_LIBRARY_DIR "${CMAKE_BINARY_DIR}")
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/setup.py ${CMAKE_CURRENT_BINARY_DIR}/setup.py)
 add_custom_command(TARGET PythonInstall
-    COMMAND "${PYTHON_EXECUTABLE}" setup.py install
+    COMMAND "${PYTHON_EXECUTABLE}" -m pip install .
```

Comment:
Allows building conda packages for newer python versions on OSX, see https://github.com/conda-forge/openmm-atmmetaforce-plugin-feedstock/blob/e8bdfa195151968c7336155b7f1fac98edc0ccfd/recipe/patches/patch-install-no-deps-osx.patch